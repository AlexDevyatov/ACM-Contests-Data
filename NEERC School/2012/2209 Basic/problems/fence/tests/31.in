8
-66 45
-65 -54
-47 -63
35 -64
65 -53
65 59
-45 69
-51 68
4 -5
