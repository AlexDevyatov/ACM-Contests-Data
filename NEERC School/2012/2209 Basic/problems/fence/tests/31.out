9
-66 45
-65 -54
-47 -63
35 -64
65 -53
65 59
-45 69
4 -5
-51 68
